pkg/composer/composer.go

```diff
@@ -49,6 +59,101 @@ type Options struct {
 	Experimental     bool // enable experimental features
 }
 
+func GetComposer(client *containerd.Client, globalOptions *types.GlobalCommandOptions, options Options, stdout, stderr io.Writer) (*Composer, error) {
```

Comment:
```suggestion
func New(client *containerd.Client, globalOptions *types.GlobalCommandOptions, options Options, stdout, stderr io.Writer) (*Composer, error) {
```

See also https://go.dev/doc/effective_go#Getters

Reply:
`New` already exists, so maybe this should be moved to `pkg/cmd/...`